semgrep_output_v1.atd

```diff
@@ -713,6 +712,17 @@ type cli_timing <ocaml attr="deriving show"> = {
     ?max_memory_bytes : int option;
   }
 
+type contribution <ocaml attr="deriving show"> = {
+    commit_hash: string;
+    commit_timestamp: string;
+    commit_author_name: string;
```

Comment:
maybe this could have been split in a separate contributor type, like it was done in the Python code of shapski?

Reply:
Good call! Added in latest commit